common/e2e_config/e2e_config.go

```diff
@@ -138,6 +138,7 @@ type ProductSpec struct {
 	LvmEnginePluginDriverName         string            `yaml:"lvmEnginePluginDriverName"`
 	UmbrellaOpenebsHelmChartName      string            `yaml:"umbrellaOpenebsHelmChartName"`
 	UseUmbrellaOpenEBSChart           bool              `yaml:"useUmbrellaOpenEBSChart" env:"e2e_use_umbrella_openebs_chart" env-default:"false"`
+	DisableLocalEngines           	  bool              `yaml:"disableLocalEngines" env:"e2e_disable_local_engines" env-default:"true"`
```

Comment:
Please default the value to false, so that if environment variable is set local engines are installed - which should be the default behaviour.

Reply:
Modified as suggested